documentation/modules/configuring/ref-storage-tiered.adoc

```diff
@@ -0,0 +1,48 @@
+[id='ref-tiered-storage-{context}']
+= Tiered storage (preview)
+
+[role="_abstract"]
+Tiered storage introduces a flexible approach to managing Kafka data whereby log segments are moved to a separate storage system. 
+For example, you can have fast local disks on brokers and offload older or less frequently accessed data from the local disks to more cost-effective, scalable remote storage solutions, such as Amazon S3, without compromising data accessibility and durability.
+
+NOTE: Tiered storage is available a technology preview with Strimzi, so it is not suitable for production environments.
+ 
+Tiered storage requires an implementation of Kafka's `RemoteStorageManager` interface, which is enabled through configuration of the `Kafka` resource.
+Strimzi uses Kafka's `TopicBasedRemoteLogMetadataManager` for Remote Log Metadata Management (RLMM) when custom tiered storage is enabled.
+The RLMM manages the metadata related to remote storage.  
+
+To use custom tiered storage, do the following:
+
+* Include a tiered storage plugin for Kafka in the Strimzi image by building a custom container image. 
+The plugin must provide the necessary functionality for a Kafka cluster managed by Strimzi to interact with the tiered storage solution.
+* Create a custom implementation of Kafka's `RemoteStorageManager` interface for your chosen tiered storage solution. 
```

Comment:
This sounds like you need to both include a plugin and implement a custom RemoteStorageManager. The plugin is the RemoteStorageManager implementation.

Reply:
Yeah, I would remover this point. If you are implementing custom tiered storage plugin, you don't need this explanation for that.